src/mgr/DaemonServer.cc

```diff
@@ -2556,11 +2556,32 @@ bool DaemonServer::_handle_command(
     return true;
   }
 
+  auto finisher_val = py_modules.get_threshold_val(mod_name);
+  auto queue_len = py_modules.get_mod_finisher_queue_len(mod_name);
+  ss << "The value of 'max_command_queue_length' config option for Module '" << mod_name << "' is "
+        << finisher_val << " and the current request queue length is " << queue_len;
+  dout(10) << ss.str() << dendl;
+
+  if (finisher_val != "0")  {
+    if (std::to_string(queue_len) > finisher_val)
```

Comment:
why are we comparing strings here ?

Reply:
@neesingh-rh ping

Reply:
Sorry, this comment was missed. Will check and update